### ISSUE TITLE
errors for Italian

### ISSUE THREAD

**Issue by gcelano** [dmort27/epitran#62]
```
epi = epitran.Epitran("ita-Latn") 
epi.transliterate("motorizzazione") 
```

return 'motorit͡sasione', but it should be, at least, 'motorit͡sat͡sione' or, better,  'motorit͡st͡sat͡st͡sione'

**Comment by dmort27** [comment]
I agree that the <z> should re realized as an affricate. This is a bug.

In the interpretation of IPA used in Epitran, geminates are represented as consonants plus the long mark, so geminate [t͡s] would be [t͡sː] rather than [t͡st͡s]. However, it is my understanding that <zz> does not represent a geminate (any longer) and I don't see why <z> would either. I agree that [i] should be [j]. This would probably be relatively easy to fix if I could find the exact rule.

Can you provide more test cases so that I can insure that Epitran is functioning properly for Italian?
